assets/js/script.js

```diff
@@ -127,6 +127,61 @@ function form() {
   var emailInputElement = document.querySelector('#email');
   var messageInputElement = document.querySelector('#message');
   var errorMessagesElement = document.querySelector('.error-messages');
+
+  class FormData {
+    constructor(formData) {
+      this.name = formData.name;
+      this.email = formData.email;
+      this.message = formData.message;
+    }
+
+    get(key) {
+      return this[key];
+    }
+
+    set(key, value) {
+      this[key] = value;
+      return this;
+    }
+
+  }
+
+  var formData = {
+    name: '',
+    email: '',
+    message: ''
+  };
+
+  var getFormData = formData => {
+    var formFromLocalStorage = localStorage.getItem('formData');
+    var result;
+
+    try {
+      result = JSON.parse(formFromLocalStorage);
+
+      if (!result) {
+        result = formData;
+      }
+    } catch (error) {
+      result = formData;
+    }
+
+    return new FormData(result);
+  };
+
+  var currentFormData = getFormData(formData);
```

Comment:
- When loading the page the inputs are not pre-filled with the stored data in the local storage, Kindly implement that
![image](https://user-images.githubusercontent.com/75126481/141371864-41ad315c-2204-4915-b249-80a77b0ff4fc.png)
- This could be helpful :point_down: 
![image](https://user-images.githubusercontent.com/75126481/141372696-4bdef13e-1a23-4876-b063-c8441f3a8dd1.png)

Reply:
Did you check the load because the file is implemented as a module and it's working.

Reply:
- Your data are being stored but when I load the page they do not pre-fill the inputs fields

Reply:
- On line 54 of your `index.html` Kindly Put the correct directory of your script 
![image](https://user-images.githubusercontent.com/75126481/141377095-4fa0f40e-aa4b-4761-8a3d-7ffc42eb53fe.png)
- It should be something like this `<script src="src/js/script.js" ></script>`

Reply:
Thank you for the comment, the src is `assets/js/script.js` because it's the compiled version of the script.